applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -715,6 +748,11 @@ Building the application on nRF5340 DK with nRF7002 EK shield
 
 .. include:: /includes/matter_building_nrf5340dk_70ek
 
+Building the application on nRF54LM20 DK with nRF7002 EB II shield
+==================================================================
```

Comment:
```suggestion
Building the application on nRF54LM20 DK with nRF7002-EB II shield
===================================================================
```

Reply:
Sorry for that, I fixed this one and the one below as well